Gems/Atom/RPI/Code/Source/RPI.Public/Material/Material.cpp

```diff
@@ -59,7 +61,10 @@ namespace AZ
             AZ_TRACE_METHOD();
 
             m_materialAsset = { &materialAsset, AZ::Data::AssetLoadBehavior::PreLoad };
+            while (m_materialAsset.GetStatus() != AZ::Data::AssetData::AssetStatus::Ready)
+            {
 
+            }
```

Comment:
How can we guarantee that a request was made to load the material asset? Should this attempt to load it? Concerned you might silently get stuck in an infinite loop here.